docs/example-scenario/apps/hpc-saas.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Computer-aided engineering through high performance computing on Azure
+description: <Article Description>
+author: alexbuckgit
+ms.date: 08/22/2018
+---
+
+# Computer-aided engineering through high performance computing on Azure
+
+This example scenario demonstrates delivery of a software-as-a-service (SaaS) platform built on the high-performance computing (HPC) capabilities of Azure. This scenario is based on an engineering software solution. However, the architecture is relevant to other industries requiring HPC resources such as image rendering, complex modeling, and financial risk calculation.
+
+This example demonstrates an engineering software provider that delivers computer-aided engineering (CAE) applications to engineering firms and manufacturing enterprises. CAE solutions enable innovation, reduce development times, and lower costs throughout the lifetime of a product's design. These solutions require a substantial compute resources and often process high data volumes. The high costs of an on-premises HPC appliance or high-end workstations often put these technologies out of reach for small engineering firms, entrepreneurs, and students. 
+
+The company wants to expand the market for its applications by building a SaaS platform backed by cloud-based HPC technologies. Their customers should be able to pay for compute resources as needed and access massive computing power that would be unaffordable otherwise. The company's goals include:
+* Taking advantage of HPC capabilities in Azure to accelerate the product design and testing process
+* Using the latest hardware innovations to run complex simulations, while minimizing the costs for simpler simulations  
+* Enabling true-to-life visualization and rendering in a web browser, without requiring a high-end engineering workstation
+
+## Potential use cases
+
+Other scenarios using this architecture might include:
+
+* Genomics research
+* Weather simulation
+* Computational chemistry applications
+
+## Architecture
+
+![Architecture for a SaaS solution enabling HPC capabilities][architecture]
+
+* Users can access NV-series virtual machines (VMs) via a browser with a HTML5-based RDP connection using the Apache Guacamole service (http://guacamole.apache.org/). These VM instances provide powerful GPUs for rendering and collaborative tasks. Users can edit their designs and view their results without needing access to high-end mobile computing devices or laptops. The Altair PBSWorks or open-source PBS Professional scheduler spins up additional nodes based on user-defined heuristics and could alternatively leverage a solution based on Azure functions.
```

Comment:
Embed link in the text

Reply:
Move the mention of Azure Functions to "alternatives" section? Also, capitalize Azure Functions

Reply:
a HTML5 > an HTML5

Reply:
"The Altair PBSWorks or open-source PBS Professional scheduler" - should this be a separate bullet point?

Reply:
Fixed. Removed the Azure Functions line b/c it seemed like a non-sequitur.